app/assets/stylesheets/custom.css.scss

```diff
@@ -133,6 +133,13 @@ a:hover, a:active {
     margin-bottom: 20px;
 }
 
+.btn-details {
+    background-color: $white;
+    border-color: $clam-tan;
+    margin-top: 10px;
+    margin: 5px;
+    color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces